app/gimlet-rot/app-b.toml

```diff
@@ -134,6 +134,14 @@ pins = [
 ]
 spi_num = 5
 
+[tasks.dumper]
```

Comment:
Should this be added in `app-c.toml` too? (Likewise for `dump_agent` on the SP)

Reply:
Yes, done -- and tested!